### DISCUSSION TITLE
sbins

### DISCUSSION THREAD

**Discussion by hanahg** [MDAnalysis/mdanalysis#4357] [Q&A]
Hi,

how to choose the best value for sbins value? 
Is there any parameters or anything should I note for choosing bins?

Thanks in advanced

**Comment by RMeli** [comment]
Hi @hanahg. Can you please give more details? What are you trying to achieve? I think we need more context in order to be able to properly help you.

**Reply by orbeckst** [reply]
That's not very helpful for us, I don't think that this is sufficient information to help. For instance, you _might_ be using InterRDF but I am not sure and I don't have the time to guess or write a long answer based on assuming what you might be using so it's on you to be precise and tell us exactly what you're doing.

You can start by showing a few (less than 10) lines of code that you are using. Then explain what you want to achieve with this code.

There are https://www.catb.org/esr/faqs/smart-questions.html and https://www.chiark.greenend.org.uk/~sgtatham/bugs.html that describe in detail good strategies to get technical answers.
